### PR TITLE
bigloo: use brewed openssl, audit fixes

### PR DESCRIPTION
**Pull request by apjanke**

### CODE REVIEW

File: Library/Formula/bigloo.rb [hunk 1]
```diff
@@ -45,4 +48,8 @@ def install
     manpages = %w[bgldepend bglmake bglpp bgltags bglafile bgljfile bglmco bglprof]
     manpages.each { |m| man1.install "manuals/#{m}.man" => "#{m}.1" }
   end
+
+  test do
+    assert_match "Bigloo (4", shell_output("bigloo --help")
```
Comment (xu-cheng) [inline Library/Formula/bigloo.rb]: Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

Comment (apjanke) [inline Library/Formula/bigloo.rb]: Sure, but it may take me a couple days to find time to do so. Want to wait on merging this until I can add it?

**Comment by DomT4** [comment]
It looks like there's an update available here:

> The source file distribution that can be used for Unix systems and Win32/Cygwin is:
> bigloo4.2a.tar.gz

**Comment by apjanke** [comment]
The 4.2a distribution is failing to compile for me on 10.9.5; may require formula tweaks.

**Comment by DomT4** [comment]
Will take this as-is to resolve the OpenSSL situation, but if you wanted to chase it in at some point with an improved test and an attempt at the version bump we certainly wouldn't object :wink:.

Merged in https://github.com/Homebrew/homebrew/commit/89ee43edaa63d3b656e83828f3a3b6beab3a1492. Thanks again @apjanke!

**Pull request closed without merging**